monkey/monkey_island/cc/services/authentication_service/flask_resources/login.py

```diff
@@ -37,6 +38,9 @@ def post(self):
         try:
             username, password = get_username_password_from_request(request)
             response: ResponseValue = login()
+            # TODO send these back
+            _tokens = self._authentication_facade.generate_refresh_token(current_user)
```

Comment:
NIT: This can get fixed in the next PR, but 
```suggestion
            refresh_token = self._authentication_facade.generate_refresh_token(current_user)
```